cli/cli_set_analytics.go

```diff
@@ -3,6 +3,7 @@ package cli
 import (
 	"context"
 	"fmt"
+	"os"
```

Comment:
Ignore the changes in this file, this is just dummy implementation to connect the dots. The actual implementation of the consent flow will be delivered here https://github.com/NordSecurity/nordvpn-linux/pull/904/files